matchers.md

```diff
@@ -192,14 +192,6 @@ it("passes if $closure echoes the expected regex output", function() {
 });
 ```
 
-```php
-it("passes if $actual matches the $expected closure logic", function() {
-    expect('Hello World!')->toMatchEcho(function($actual) {
```

Comment:
I haven't validated it yet as I'm still getting up to speed but my .02 here is that, if `toMachEcho` is able to accept a function closure as an argument it may be preferable to keep this example even though it throws an error as an example of the power of closures and procedural/functional programming paradigms being pulled into this wonderful testing library from the world of JS (or possibly Elm, depending on where they came from).